test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -470,7 +480,7 @@ def test_avs_failed_authorize
   def test_invalid_shipping_address
     response = @gateway.authorize(@amount, @credit_card, shipping_address: address.update(country: 'Canada'))
     assert_failure response
-    assert_equal 'request_invalid: address_country_invalid', response.message
+    assert_equal 'request_invalid: country_address_invalid', response.message
```

Comment:
Thanks for fixing these! Do you, uh, want to fix the other 3 as well? If you're getting the same failures I'm getting, I think those are all test expectations that can be updated

Reply:
Haha sure! I was working on this way too late at night so I was probably just too tired to notice.